app/app/controllers/report_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ReportController < ApplicationController
   def index
-    @reports = DailyLog.where("EXTRACT(MONTH FROM day) = 9").joins("INNER JOIN employees ON employees.status = true").to_a.count -(Employee.where("status = true").to_a.count*Time.days_in_month(Time.current.month))
+  	@active_employees = Employee.where("status = true")    
+    @monthly_logs = DailyLog.where("EXTRACT(MONTH FROM day) = 9").joins(:employee ).where(" employees.status = true ")
+    @reports = @monthly_logs.count -(@active_employees.count*Time.days_in_month(Time.current.month))
+    @employeeday = @monthly_logs.where(" EXTRACT(DAY FROM day) = 14 ").count - @active_employees.count
```

Comment:
This method can be refactor, separate it to not introduce all this logic in one method.

Reply:
resolve 👍